### ISSUE TITLE
Multi-touch gesture

### ISSUE THREAD

**Issue by RiccardoPP** [Rezi/svelte-gestures#4]
Hi, thanks for the code.

I'm a newbie. I'd like to recognize a two-fingers tap con the screen. Is it possible? I'll use it to open a modal.

Thanks,

**Comment by Rezi** [comment]
Hi Riccardo, here is custom gesture I made with the low level api of svelte-gestures: 
https://svelte.dev/repl/ddb074827a834f828773ee869d0a2d91?version=3.38.2

you can copy the code from there. Once I have time to test it a bit, I'll add it as new gesture to the library

**Comment by RiccardoPP** [comment]
Thanks, I'll try it and let you know.

**Comment by Rezi** [comment]
Have you successfully used it @RiccardoPP ?

**Comment by RiccardoPP** [comment]
Hi Rezi, sorry but my project was halted and I haven't tested it yet.

**Comment by nooblyf** [comment]
Repl points to a 404 now, can someone share how to do it? or is this planned?

**Comment by nooblyf** [comment]
Repl points to a 404 now, @Rezi can you share it again?

**Comment by Rezi** [comment]
Thanks for notification @nooblyf. Looks like I missed that repls have been migrated to playground path:
Here it is still available 
https://svelte.dev/playground/ddb074827a834f828773ee869d0a2d91?version==5.16.2
Let me know if it works well. If so I'll make it part of the gestures library ;)

**Comment by nooblyf** [comment]
Thanks @Rezi this works! it doesn't update co-ordinates for one & two finger input as expected but there's a bug because it also updates co-ordinates for more than 3 fingers (4 & 5)

I tested it on an iPhone 12 `iOS 18.2` - Safari and Chrome `v131.0.6778.154`

**Comment by nooblyf** [comment]
i edited that repl and checked the logs in [this](https://svelte.dev/playground/ddb074827a834f828773ee869d0a2d91?version=5.16.2#H4sIAAAAAAAAE5VVbYvaQBD-K8NyYKTbnNfSL9EIxbaf-gZ3_WSEyyWjLo27YXeiCSH_vWwS8-JpaREx7sw888yzM5OSyfCAzGOPR0wIYYeGMo3Gg0OWkABSWbRnnG1FgoZ565JRkVp_e8D4OfpjmrqmRmCcvYQGr51HShJKMsxjCxNpkdIykJPMIBjSIqLJPJDikCpNUIJB-qmEJNQrJUmrxHDYIa3QHv3YPp1UbTZQwVarA0yaNG_PBViwQGJew20zGZFQsinqydbkSBUjhzTU4QEJtQEfyqbclcokefAOqimUgQwoUtLQWZrv4QHBh0kPZVMBJEhteE2xTg99ZiV_pU4YkTji56NVocYGABBbGBncBOWO9rAYkHN7Yl0cQCwMhTKydGSWJPPmvLI_leXdfsc0PqmTfE0koKZMlaCbqJ3zfFde4VSB7_twV14lVj1P53-vyAbfrslSaDnTWfMhyhelvyqVgg9r13WHFj7yW882m_kYJlJKx-YxO4A_cnU1xlmEjhNGUTZC4SBkjPnUX_bMaHC94F_txrar1kOkKzWsa3B4Aw-bzXTeJ7A03Bze-MNOcvNLj-LSoxh4aKRMS7CO3WnFy9ybcSi8WTXIN9YIm2GzStlZ6BFzD76FtHe1ymTsdFq6OdzDlWue8j60uBFa3AjtCF9ytLq6sTBpSNG-jnHOUwAg8QSrzJA6NJbBpPJ-WuqJQQpF4kHZluuNyuZAod4heYAWx23-QcV7iGp6fh7qOBo7CdDewesdNqbWNIsdXd4OJq_3hIXuFfiHD_yvc7uvTBQm2G2vvHsqLrfXPpRxgtqpdek2UB0OfqtWo63bYwLkl8ao72hrL27Yi3kr5uK-e1EEchGLoz3PDHr99vXL8dp-D1V9FUoOnQJWtjVUAbNmQ0WCfsBOIqa992E2S_P5HsVuT-2fF6Vj1N5DmoNRiYjhJQmj33MbvWzUGbwhu3bKocwrDgWURU0_Fscl44wwJ-aRzrDaVH8AtnbgcXQHAAA=), it seems like four separate events are fired when i do a four finger touch for instance, that's why it eventually passes through in this if check
```js
if (activeEvents.length === parameters.touchCount) {
```

<img src="https://github.com/user-attachments/assets/6d973748-8427-4d66-b078-0635a4d21950" width="480px">

**Comment by Rezi** [comment]
right, the repl was just a simple example. It makes sense to add some input parameter for the gesture with count and only send the event when the count is reached. Will add this option to the gesture and then add it to the library. Thanks for feedback
